src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/resource/ExampleResource.kt

```diff
@@ -5,19 +5,19 @@ import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
-import uk.gov.justice.digital.hmpps.hmppstemplatepackagename.service.TemplateKotlinApiService
+import uk.gov.justice.digital.hmpps.hmppstemplatepackagename.service.ExampleApiService
 import java.time.LocalDateTime
 
 // This controller is expected to be called from the UI - so the hmpps-template-typescript project.
 @RestController
 // Role here is specific to the UI.
 @PreAuthorize("hasRole('ROLE_TEMPLATE_KOTLIN__UI')")
 @RequestMapping(value = ["/example"], produces = ["application/json"])
-class ExampleResource(private val templateKotlinApiService: TemplateKotlinApiService) {
+class ExampleResource(private val exampleApiService: ExampleApiService) {
 
   @GetMapping("/time")
-  fun getTime(): LocalDateTime = LocalDateTime.now()
+  fun getTime(): LocalDateTime = exampleApiService.getTime()
 
-  @GetMapping("/user-message/{user}")
-  fun getUserMessage(@PathVariable user: String) = templateKotlinApiService.getUserMessage(user)
+  @GetMapping("/message/{parameter}")
```

Comment:
changed the endpoint not to refer to a `user` as this suggested that the user was being passed through which would be bad practise as the user would then be in the url